### ISSUE TITLE
Maven dependency resolution may not work correctly when multiple repositories are listed in the pom file.

### ISSUE THREAD

**Issue by tkvangorder** [openrewrite/rewrite#166]
If a maven file has multiple repositories defined, rewrite will only attempt to download the pom.xml from the first repository. Rewrite caches the result (including a miss) and will not consult the additional repositories.

**Comment by sambsnyd** [comment]
This was fixed in #165 
@tkvangorder if your commit messages include a phrase like "fixes #166" or "closes #166" github will automatically close the issue once the commit is merged into master. Saves having to manually close such issues after.
